tests/integration/seccomp.bats

```diff
@@ -33,3 +33,71 @@ function teardown() {
 	runc run test_busybox
 	[ "$status" -eq 0 ]
 }
+
+# TODO:
+# - Test other actions like SCMP_ACT_TRAP, SCMP_ACT_TRACE, SCMP_ACT_LOG.
+# - Test args (index, value, valueTwo, etc).
+
+@test "runc run [seccomp] (SCMP_ACT_ERRNO default)" {
+	update_config '.process.args = ["/bin/sh", "-c", "mkdir /dev/shm/foo"] |
+		.process.noNewPrivileges = false |
+		.linux.seccomp = {
```

Comment:
Not matters much, but elsewhere in tests/integration we usually use slightly different format of multiline `update_config` statements. See e.g.
https://github.com/opencontainers/runc/blob/9835e9c6b2959ccf40c23c215202f9a9b1d9644c/tests/integration/cgroups.bats#L169-L173

(the first line is tab and two spaces, the rest of them just tabs and `| ` etc.)

Reply:
Fixed the style, thanks!